### PR TITLE
Switch crappy MySQL to Postgresql

### PR DESCRIPTION
**Pull request by TomasVotruba**
- [x] prepare migration SQL inserts 
- [x] update docker image

### CODE REVIEW

**Comment by JanMikes** [comment]
Just fyi, if you want to move to postgre as default driver, be careful as mysql 5.7 is running on production (in docker) and i am not sure about 100% compatibility between postgre and mysql, for someone migrations might not work.

**Comment by JanMikes** [comment]
Said that, application can not support 2 different database drivers, it either has to use mysql driver or pgsql driver, never can do both

**Comment by TomasVotruba** [comment]
1 db driver is clear, this is just prepare for the jump :) from one to another, so I kept both untill it's all working.

I've tried to update some docker stuff. Could you review?

File: .docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ if [ "$1" = 'apache2-foreground' ] || [ "$1" = 'bin/console' ] || [ "$1" = 'php'
     ## Check if variable DATABASE_HOST is set, if yes, we have database
     if [[ -v DATABASE_HOST ]]; then
         ## Wait until database connection is ready
-        until mysql -u $DATABASE_USER -h $DATABASE_HOST --password="$DATABASE_PASSWORD" -e "" ; do
+        # @todo needs update
+        until postgres -u $DATABASE_USER -h $DATABASE_HOST --password="$DATABASE_PASSWORD" -e "" ; do
```
Comment (TomasVotruba) [inline .docker/docker-entrypoint.sh:21]: What whould be here? I haven't use postgres in CLI manner at all.

**Comment by TomasVotruba** [comment]
Looks like some cache issues like last time. I'll try to merge it and see if that helps

Prepare for hotfix PRs :D

File: .docker/docker-entrypoint.sh (hunk 1 above)
Comment (JanMikes) [reply .docker/docker-entrypoint.sh:21]: me neither, i am really sceptic to this migration, to technology none of us both can maintain at good grade

**Pull request merged**